help/en/html/tools/logixng/LogixNGTableTableEditor.shtml

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
+
+<html lang="en">
+<head>
+
+<!-- Copyright Dave Sand 2021 -->
+
+  <title>JMRI: LogixNG Table Editor</title>
+  <meta name="author" content="Dave Sand">
+  <meta name="keywords" content="JMRI logixng tables editor">
+
+  <!--#include virtual="/Style.shtml" -->
+
+</head>
+
+<body>
+  <!--#include virtual="/Header.shtml" -->
+  <div class="nomenu" id="mBody">
+    <div id="mainContent">
+
+      <h1>JMRI: LogixNG Table Editor</h1>
```

Comment:
It might be good to have this `<h1>JMRI: LogixNG Tables Table Editor</h1>` to make it clear that this is not about the ordinary LogixNG Table Editor.

But if you prefer `<h1>JMRI: LogixNG Table Editor</h1>`, then I'm fine with that too.

Reply:
Done.